### PR TITLE
Add `slim` versions for `python`

### PR DESCRIPTION
**Pull request by IronicUsername**
Closes `my eyes`

### Description
- make `3.8-slim` & `3.9-slim` versions of the `3.8`/`3.9` `Dockerfile`

Reason:
@deusavalon described in `file-conversions` issue [#49](https://github.com/sidestream-tech/file-conversions/issues/49) that the docker container  was [too big](https://www.youtube.com/watch?v=fzdg5OyXVs8) after creation.
At that point we used `sidestream/python-poetry:3.9`.
I thought it would be cool if we also had a slim version of our `python-poetry` images..

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,20 +3,21 @@
...
-    - `helm` is used to deploy the application
-    - image name: `sidestream/aws-cli-chamber-helm`
-- `ci-cd/aws-cli-chamber-helmsman`: `aws-cli` + `chamber`  + `helm` + `helmsman`  images and files in order to support our continuous delivery, where:
-    - `aws-cli` is used to connect to AWS infrastructure
-    - `chamber` is used to securely inject secrets into the deployment,
-    - `helm` is used to deploy the application
-    - `helmsman` is used as a desired state file tooling for helm
-    - image name: `sidestream/aws-cli-chamber-helmsman`
+
+- `python-poetry`: Where `poetry` is the amazing [python package manager](https://python-poetry.org/). So far other images we found were outdated and unmaintained. We wanted to change that, as we believe in `poetry` as the future of python package management and thats why are we using [`acidrain`'s](https://hub.docker.com/u/acidrain) `Dockerfile`s [`python-poetry`](https://hub.docker.com/r/acidrain/python-poetry). They are very good 🙂
```
Comment (BracketJohn) [inline README.md:7]:
```suggestion
- `python-poetry`: Where `poetry` is the amazing [python package manager](https://python-poetry.org/). So far other images we found were outdated and unmaintained. We recommend to use [`acidrain`'s](https://hub.docker.com/u/acidrain) `Dockerfile`s [`python-poetry`](https://hub.docker.com/r/acidrain/python-poetry). They are very good 🙂
```

**Review by BracketJohn — COMMENTED** [review]
please rever changing all formatting of the readme to indentation 2 as well, that makes the change buigger than it needs to be (:

**Comment by BracketJohn** [comment]
yeah, but its your personal vs code settings that are applied, so if you configure for example 2 spaces then a change would not happen!

**Comment by IronicUsername** [comment]
ah got it. i _think_ i did not change smth there. but since this can be configured by other peeps you are making a valid point. 
I thought the vscode formater formats after some "official" rules.

**Pull request merged**